src/server/server.ts

```diff
@@ -262,6 +288,8 @@ app.get('/*', bodyParser.raw(), serveArticleGet);
 
 app.post('/article', bodyParser.raw(), serveArticlePost);
 
+app.post('/render-edition-article', bodyParser.raw(), serveEditionArticlePost);
```

Comment:
Perhaps name the endpoint this for consistency with the MAPI endpoint?

```suggestion
app.post('/editions-article', bodyParser.raw(), serveEditionArticlePost);
```